capirca/aclgen.py

```diff
@@ -53,61 +53,76 @@
 from capirca.lib import srxlo
 from capirca.lib import windows_advfirewall
 
+from capirca.utils import config
+
 
 FLAGS = flags.FLAGS
 
 
 def SetupFlags():
   flags.DEFINE_string(
-      'base_directory',
-      './policies',
-      'The base directory to look for acls; '
-      'typically where you\'d find ./corp and ./prod')
+    'base_directory',
+    None,
```

Comment:
It seems you've changed all the defaults to None; would this potentially cause problems with people omitting a flag because they wanted the default value?

Reply:
The functionality is maintained through the configs.default object. Down below in the main function, there is a line that extracts just the set values and applies them over the defaults, and the defaults are preserved as the same as before